orca/orca-queue/src/test/java/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerExceptionHandlerTest.java

```diff
@@ -308,16 +250,21 @@ void taskThrowsSpinnakerHttpExceptionNoRetry(SpinnakerHttpException spinnakerHtt
   }
 
   private static Stream<SpinnakerHttpException> nonRetryableSpinnakerHttpExceptions() {
-    return nonRetryableRetrofitErrors()
-        .map(retrofitError -> makeSpinnakerHttpException(retrofitError));
+    // irrespective of the exception details, http code 500 is non-retryable
```

Comment:
It seems a little strange that orca doesn't retry GETs/idempotent requests on 500, but it does look like that's the way it's implemented.